napari/_qt/widgets/_tests/test_qt_play.py

```diff
@@ -37,7 +37,7 @@ def bump(*args):
             worker._stop()
 
     def count_reached():
-        assert worker._count >= nframes
+        return worker._count >= nframes
```

Comment:
Using `assert` over `return` is preferred due to the associated failure message: https://pytest-qt.readthedocs.io/en/latest/wait_until.html#waituntil-waiting-for-arbitrary-conditions

Is there a reason you made the change?

Reply:
I just assumed it was unintentional cause I've never seen something like this, but I should have checked. I'll revert!